dpx/src/dpx_pdfdev.rs

```diff
@@ -134,35 +133,101 @@ impl pdf_tmatrix {
 }
 #[derive(Copy, Clone, Default)]
 #[repr(C)]
-pub struct pdf_rect {
-    pub llx: f64,
-    pub lly: f64,
-    pub urx: f64,
-    pub ury: f64,
+pub struct Rect {
+    pub ll: Coord,
```

Comment:
The tiniest bit of documentation on these cryptic little initialisms would go a long way